targets/TARGET_Samsung/TARGET_SIDK_S5JS100/BINARIES/README.md

```diff
@@ -0,0 +1,19 @@
+Explanation of files
+* pbl_dcache_jumpto_mbedos.bin: 
```

Comment:
These binaries will need license files, reference https://os.mbed.com/docs/mbed-os/v5.12/contributing/license.html. It needs license files to be added